Kickstarter-iOS/Views/Cells/RewardCell.swift

```diff
@@ -23,6 +23,17 @@ final class RewardCell: UICollectionViewCell, ValueCell {
   private let descriptionLabel = UILabel(frame: .zero)
   private let descriptionStackView = UIStackView(frame: .zero)
   private let descriptionTitleLabel = UILabel(frame: .zero)
+  private let imageView: UIImageView = {
```

Comment:
That’s exactly why I tried to keep it generic (because it’s either checkmark or exclamation mark) but let me try thinking about better name. 

Maybe `stateImageView`, `promptImageView`, `statusImageView` ¯\_(ツ)_/¯

Reply:
Oh I see! Well the rename to `stateImageView` is cool. Thanks for making those changes!